build/StripSymbols.ps1

```diff
@@ -0,0 +1,80 @@
+﻿param(
+    [parameter(mandatory)][string] $InputDirectory,
+    [parameter(mandatory)][string] $OutputDirectory
+)
+
+$ErrorActionPreference = "Stop"
+function Get-Is64Bit
+{
+    return $env:PROCESSOR_ARCHITECTURE -eq "AMD64"
+}
+
+function Get-DebuggingToolsRoot
+{
+    # Constants
+    $windowsSDKRegPath = if (Get-Is64Bit) { "HKLM:\Software\WOW6432Node\Microsoft\Windows Kits\Installed Roots" } else { "HKLM:\Software\Microsoft\Windows Kits\Installed Roots" }
+    $windowsDebuggingToolsRegRootKey = "WindowsDebuggersRoot10"
+
+    try
+    {
+        return Get-ItemProperty -Path $windowsSDKRegPath | Select-Object -ExpandProperty $windowsDebuggingToolsRegRootKey
+    }
+    catch
+    {
+        return $null
+    }
+}
+
+function Strip-PrivateInformation(
```

Comment:
Nit: `Strip` isn't an [approved PowerShell verb](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands).
```suggestion
function Remove-PrivateSymbolInformation(
```